### ISSUE TITLE
Stuck on Core version 0.105.5 no update available in UI or CLI

### ISSUE THREAD

**Issue by nhorvath** [home-assistant/supervisor#1630]
**Home Assistant release with the issue:**

```
arch | x86_64
-- | --
dev | false
docker | true
hassio | true
os_name | Linux
python_version | 3.7.6
timezone | America/New_York
version | 0.105.5
virtualenv | false
```

**Operating environment (HassOS/Generic):**

Virtualbox VM running HassOS

**Supervisor logs:**

21-01-22 04:42:31 INFO (MainThread) [hassio.utils.gdbus] Connect to dbus: org.freedesktop.hostname1 - /org/freedesktop/hostname1
21-01-22 04:42:31 INFO (MainThread) [hassio.utils.gdbus] Connect to dbus: de.pengutronix.rauc - /
21-01-22 04:42:31 INFO (MainThread) [hassio.utils.gdbus] Connect to dbus: org.freedesktop.NetworkManager - /org/freedesktop/NetworkManager/DnsManager
21-01-22 04:42:31 INFO (MainThread) [hassio.host.info] Update local host information
21-01-22 04:42:31 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.DBus.Properties.GetAll on /org/freedesktop/hostname1
21-01-22 04:42:31 INFO (MainThread) [hassio.host.services] Update service information
21-01-22 04:42:31 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.systemd1.Manager.ListUnits on /org/freedesktop/systemd1
21-01-22 04:42:31 INFO (MainThread) [hassio.host.network] Update local network DNS information
21-01-22 04:42:31 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.DBus.Properties.GetAll on /org/freedesktop/NetworkManager/DnsManager
21-01-22 04:42:31 INFO (MainThread) [hassio.host.apparmor] Load AppArmor Profiles: {'hassio-supervisor'}
21-01-22 04:42:31 INFO (MainThread) [hassio.host.services] Reload local service hassos-apparmor.service
21-01-22 04:42:31 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.systemd1.Manager.ReloadOrRestartUnit on /org/freedesktop/systemd1
21-01-22 04:42:31 INFO (SyncWorker_1) [hassio.docker.interface] Attach to homeassistant/amd64-hassio-dns with version 1
21-01-22 04:42:31 INFO (MainThread) [hassio.dns] Start CoreDNS plugin
21-01-22 04:42:31 INFO (SyncWorker_1) [hassio.docker.interface] Clean hassio_dns application
21-01-22 04:42:31 INFO (MainThread) [hassio.misc.forwarder] Start DNS port forwarding to 172.30.32.3
21-01-22 04:42:32 INFO (SyncWorker_1) [hassio.docker.dns] Start DNS homeassistant/amd64-hassio-dns with version 1 - 172.30.32.3
21-01-22 04:42:32 INFO (SyncWorker_2) [hassio.docker.interface] Attach to homeassistant/qemux86-64-homeassistant with version 0.105.5
21-01-22 04:42:32 INFO (MainThread) [hassio.utils.gdbus] Call org.freedesktop.DBus.Properties.GetAll on /
21-01-22 04:42:32 INFO (MainThread) [hassio.hassos] Detect HassOS 3.11 / BootSlot A
21-01-22 04:42:32 INFO (SyncWorker_0) [hassio.docker.hassos_cli] Found HassOS CLI homeassistant/amd64-hassio-cli with version 19
21-01-22 04:42:32 INFO (MainThread) [hassio.store.git] Load add-on /data/addons/git/a0d7b954 repository
21-01-22 04:42:32 INFO (MainThread) [hassio.store.git] Load add-on /data/addons/git/cebe7a76 repository
21-01-22 04:42:32 INFO (MainThread) [hassio.store.git] Load add-on /data/addons/git/15ef4d2f repository
21-01-22 04:42:32 INFO (MainThread) [hassio.store.git] Load add-on /data/addons/core repository
21-01-22 04:42:32 INFO (MainThread) [hassio.store] Load add-ons from store: 69 all - 69 new - 0 remove
21-01-22 04:42:32 INFO (MainThread) [hassio.addons] Found 13 installed add-ons
21-01-22 04:42:32 INFO (SyncWorker_2) [hassio.docker.interface] Attach to homeassistant/amd64-addon-mosquitto with version 5.1
21-01-22 04:42:32 INFO (SyncWorker_1) [hassio.docker.interface] Attach to hassioaddons/ftp-amd64 with version 3.3.1
21-01-22 04:42:32 INFO (SyncWorker_3) [hassio.docker.interface] Attach to sabeechen/hassio-google-drive-backup-amd64 with version 0.100.0
21-01-22 04:42:32 INFO (SyncWorker_7) [hassio.docker.interface] Attach to homeassistant/amd64-addon-configurator with version 4.7
21-01-22 04:42:32 INFO (SyncWorker_2) [hassio.docker.interface] Attach to homeassistant/amd64-addon-ssh with version 8.5.3
21-01-22 04:42:32 INFO (SyncWorker_8) [hassio.docker.interface] Attach to hassioaddons/traccar-amd64 with version 0.7.3
21-01-22 04:42:32 INFO (SyncWorker_5) [hassio.docker.interface] Attach to homeassistant/amd64-addon-letsencrypt with version 4.7
21-01-22 04:42:32 INFO (SyncWorker_6) [hassio.docker.interface] Attach to hassioaddons/phlex-amd64 with version 1.2.0
21-01-22 04:42:32 INFO (SyncWorker_4) [hassio.docker.interface] Attach to homeassistant/amd64-addon-samba with version 9.0
21-01-22 04:42:32 INFO (SyncWorker_1) [hassio.docker.interface] Attach to hassioaddons/unifi-amd64 with version 0.14.1
21-01-22 04:42:32 INFO (SyncWorker_3) [hassio.docker.interface] Attach to hassioaddons/sonweb-amd64 with version 0.10.0
21-01-22 04:42:32 INFO (SyncWorker_7) [hassio.docker.interface] Attach to esphome/esphome-hassio-amd64 with version 1.14.3
21-01-22 04:42:32 INFO (SyncWorker_9) [hassio.docker.interface] Attach to hassioaddons/motioneye with version 0.8.0
21-01-22 04:42:32 INFO (MainThread) [hassio.updater] Fetch update data from https://version.home-assistant.io/stable.json
21-01-22 04:42:32 WARNING (MainThread) [hassio.updater] Can't fetch versions from https://version.home-assistant.io/stable.json: Cannot connect to host version.home-assistant.io:443 ssl:None [Address not available]
21-01-22 04:42:32 INFO (MainThread) [hassio.snapshots] Found 8 snapshot files
21-01-22 04:42:32 ERROR (MainThread) [hassio.snapshots.snapshot] Can't read snapshot tarfile /data/backup/5df22b8b.tar: unexpected end of data
21-01-22 04:42:32 INFO (MainThread) [hassio.discovery] Load 1 messages
21-01-22 04:42:32 INFO (MainThread) [hassio.ingress] Load 0 ingress session
21-01-22 04:42:32 INFO (MainThread) [hassio.secrets] Load Home Assistant secrets: 32
21-01-22 04:42:32 INFO (MainThread) [__main__] Run Hass.io
21-01-22 04:42:32 INFO (MainThread) [hassio.api] Start API on 172.30.32.2
21-01-22 04:42:32 INFO (MainThread) [hassio.utils.gdbus] Call de.pengutronix.rauc.Installer.Mark on /
21-01-22 04:42:32 INFO (MainThread) [hassio.hassos] Rauc: A - marked slot kernel.0 as good
21-01-22 04:42:32 INFO (MainThread) [hassio.addons] Phase 'initialize' start 0 add-ons
21-01-22 04:42:32 INFO (MainThread) [hassio.addons] Phase 'system' start 2 add-ons
21-01-22 04:42:32 WARNING (MainThread) [hassio.addons.validate] Unknown options ipv6
21-01-22 04:42:33 INFO (SyncWorker_4) [hassio.docker.addon] Start Docker add-on hassioaddons/sonweb-amd64 with version 0.10.0
21-01-22 04:42:33 INFO (SyncWorker_9) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-mosquitto with version 5.1
21-01-22 04:42:34 INFO (MainThread) [hassio.services.modules.mqtt] Set core_mosquitto as service provider for mqtt
21-01-22 04:42:35 INFO (MainThread) [hassio.api.security] /host/info access from a0d7b954_sonweb
21-01-22 04:42:38 INFO (MainThread) [hassio.addons] Phase 'services' start 6 add-ons
21-01-22 04:42:38 INFO (MainThread) [hassio.host.alsa] Update ALSA device list
21-01-22 04:42:38 INFO (SyncWorker_6) [hassio.docker.addon] Start Docker add-on hassioaddons/traccar-amd64 with version 0.7.3
21-01-22 04:42:39 INFO (SyncWorker_4) [hassio.docker.addon] Start Docker add-on hassioaddons/ftp-amd64 with version 3.3.1
21-01-22 04:42:39 INFO (SyncWorker_7) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-samba with version 9.0
21-01-22 04:42:39 INFO (SyncWorker_0) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-ssh with version 8.5.3
21-01-22 04:42:39 INFO (SyncWorker_5) [hassio.docker.addon] Start Docker add-on hassioaddons/phlex-amd64 with version 1.2.0
21-01-22 04:42:40 INFO (SyncWorker_9) [hassio.docker.addon] Start Docker add-on hassioaddons/unifi-amd64 with version 0.14.1
21-01-22 04:42:40 INFO (MainThread) [hassio.api.security] /host/info access from a0d7b954_traccar
21-01-22 04:42:40 INFO (MainThread) [hassio.api.security] /host/info access from a0d7b954_ftp
21-01-22 04:42:40 INFO (MainThread) [hassio.api.security] /host/info access from a0d7b954_unifi
21-01-22 04:42:41 INFO (MainThread) [hassio.api.security] /host/info access from a0d7b954_phlex
21-01-22 04:42:41 INFO (MainThread) [hassio.api.security] /dns/info access from a0d7b954_traccar
21-01-22 04:42:45 INFO (SyncWorker_1) [hassio.docker.interface] Start homeassistant/qemux86-64-homeassistant
20-04-04 11:51:47 INFO (MainThread) [hassio.homeassistant] Detect a running Home Assistant instance
20-04-04 11:51:47 INFO (MainThread) [hassio.addons] Phase 'application' start 4 add-ons
20-04-04 11:51:47 WARNING (MainThread) [hassio.addons.validate] Unknown options port
20-04-04 11:51:47 INFO (SyncWorker_4) [hassio.docker.addon] Start Docker add-on hassioaddons/motioneye with version 0.8.0
20-04-04 11:51:47 INFO (SyncWorker_6) [hassio.docker.addon] Start Docker add-on esphome/esphome-hassio-amd64 with version 1.14.3
20-04-04 11:51:48 INFO (SyncWorker_0) [hassio.docker.addon] Start Docker add-on homeassistant/amd64-addon-configurator with version 4.7
20-04-04 11:51:48 INFO (SyncWorker_1) [hassio.docker.addon] Start Docker add-on sabeechen/hassio-google-drive-backup-amd64 with version 0.100.0
20-04-04 11:51:48 INFO (MainThread) [hassio.api.security] /host/info access from a0d7b954_motioneye
20-04-04 11:51:48 INFO (MainThread) [hassio.api.security] /host/info access from 15ef4d2f_esphome
20-04-04 11:51:49 INFO (MainThread) [hassio.api.security] /dns/info access from a0d7b954_motioneye
20-04-04 11:51:50 INFO (MainThread) [hassio.homeassistant] Updated Home Assistant API token
20-04-04 11:51:50 INFO (MainThread) [hassio.api.security] /homeassistant/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:50 INFO (MainThread) [hassio.api.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots/c327895d/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots/4e40d847/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots/91eb7a7d/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots/809b2045/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots/26527348/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots/70aeecd6/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:51 INFO (MainThread) [hassio.api.security] /snapshots/4e56baec/info access from cebe7a76_hassio_google_drive_backup
20-04-04 11:51:53 INFO (MainThread) [hassio.tasks] All core tasks are scheduled
20-04-04 11:51:53 INFO (MainThread) [hassio.core] Hass.io is up and running
20-04-04 11:54:50 INFO (SyncWorker_3) [hassio.docker.interface] Restart homeassistant/qemux86-64-homeassistant
20-04-04 11:54:52 ERROR (MainThread) [hassio.homeassistant] Error on call https://172.30.32.1:8123/api/states/binary_sensor.snapshots_stale: Cannot connect to host 172.30.32.1:8123 ssl:None [Connection refused]
20-04-04 11:54:52 ERROR (MainThread) [hassio.api.proxy] Error on API for request states/binary_sensor.snapshots_stale
20-04-04 11:55:10 INFO (MainThread) [hassio.homeassistant] Detect a running Home Assistant instance
20-04-04 11:56:18 WARNING (MainThread) [hassio.homeassistant] Version 0.105.5 is already installed

**Description of problem:**
No update shows in Supervisor tab in UI.
Core update completes instantly, does nothing.
```
ha > core update
Processing... Done.

Command completed successfully.
```
```
ha > core info
arch: amd64
boot: true
custom: false
image: homeassistant/qemux86-64-homeassistant
ip_address: 172.30.32.1
last_version: 0.105.5
machine: qemux86-64
port: 8123
ssl: true
version: 0.105.5
wait_boot: 600
watchdog: true
```

**Comment by nhorvath** [comment]
So I was just reading through the logs and if you look at the timestamp when it starts up it thinks it's Jan 22 2021 and makes an unsuccessful request to https://version.home-assistant.io/stable.json then around the time it starts core time updates to correct date/time. The failed call to get version is obviously causing my issue but it appears to be caused by the date not being correct / stuff happening before the network is up.

**Comment by nhorvath** [comment]
Restarting the VM doesn't help.

**Comment by nhorvath** [comment]
I got it working, but i'm not sure what particular thing fixed it. For reference I did the following:
Shut down VM completely.
Updated Virtualbox a few minor versions.
Restarted the Virtualbox host.
Then when the VM came up with the host it started all the addons but core was inaccessible.
Tried to restart core from CLI to no avail (restart failed)
Updated hass os from 3.11 to 3.12
Updated dns addon from version 1 to 3
Restarted from CLI with host reboot.
When it came back up it was working again and the update to 107 showed.
